lang/en-EN/game.xml

```diff
@@ -2585,6 +2592,8 @@
 			<story_3><![CDATA[<b>100 Pyromancy:</b>⮀ You feel like an unstoppable wildfire, burning enemies to a crisp in an instant. You haven't had an… accident in a long time, and controlling the flames is becoming second-nature. Fire isn't the solution to every problem, but here and now you feel like it just might be.]]></story_3>
             <!--Referencing Roy Mustang from Fullmetal Alchemist.-->
             <story_4><![CDATA[<b>1,000 Pyromancy:</b>⮀ You snap your fingers idly a few times, getting a feel for the motion before turning your attention to a piece of wood you prepared. With one last snap, the entire length of the wood catches fire all at once, and within moments a cozy campfire crackles merrily before you. Mundane as such a trick may seem, making fire just from the friction between your fingers is so complex that the runes you studied only hinted that it <i>might</i> be possible. You smirk as you recall one beautifully-illustrated account you found in a side tunnel that told of Mustang, a legendary pyromancer who wore a specially-crafted pair of white gloves that would create a spark whenever he snapped his fingers, just so that he could do what you can now do with your bare hands. Truly, you have surpassed the ancient masters!]]></story_4>
+            <!-- Alternate 1,000 pyromancy story from IV:
+               The fire is conjured and extinguished in tune with your breath, an extension of your will as much as your arms and legs. In a fight you are the burning battlefield itself, every wisp of flame bends to your command and seeks only one thing: the complete incineration of your foes in a ravenous inferno. And then the battle ends and you withdraw the fire back into yourself, where it waits for another enemy. -->
```

Comment:
Hmm, it doesn't feel right to me to have a slow-paced story after a really dramatic story like that. If we're including IV's story here, I think it should be the last one in the series - what do you think of putting the Fullmetal Alchemist reference at 500? We'd just need to replace the last line with something less strong - maybe something like "Pathetic."?

Either way is fine - I don't know how attached you are to the current 1k story.

Reply:
a bit attached - I wrote it! (or, well, rewrote it to improve the FMA reference.) I was thinking about suggesting moving that to 500, but I'm still torn. I sort of like the idea of "anyone can make a big fire, but mastery is fine control" or that sort of thing, but the problem is that you're _right_, the pacing is off, and honestly I was thinking the same.

what would you think of adding a closing sting to the 1k, something that implies that it comes at the tail end of a lot of violence, like "behind you, the charred remains of the mountainside continue to smolder"? I'll leave it up to you: either amend the 1k like that to improve the pacing, move it to 500 and edit it so that you're matching rather than exceeding the ancients (and you could add a callback to Mustang in the 1k, if you wanted, "even Mustang couldn't have imagined" or similar), or just leave it alone and we'll come back to it later 😄

Reply:
I tried moving the 1k to 500, but the pacing was still off. It would be fine on its own, but next to the other pyromancy stories, I found the difference to be jarring. I ended up having to basically rewrite the story to bring the length down enough to match the others. I like how it came out, but it's kind of not what you wrote anymore (although I did get the smoldering mountainside thing in, that was nice).

If you want to keep your story in as it was before, I can revert the commit (it is your fork, after all). What do you think?

Reply:
I like it a lot! The most important thing is that my girlfriend (who introduced me to FMA) got to see the full story during her playthrough 😂 Also, the pacing works great - I like how now, 500 Pyro is where "you can do as well as the ancients if you've made yourself the tools" and 1k is just "you are the literal embodiment of fire, tools what tools"